RecoTauTag/Configuration/python/loadRecoTauTagMVAsFromPrepDB_cfi.py

```diff
@@ -7,7 +7,7 @@
     toGet = cms.VPSet(),                                             
     #  connect = cms.string("frontier://FrontierPrep/CMS_COND_PHYSICSTOOLS") # prep database
     connect = cms.string('frontier://FrontierProd/CMS_COND_PAT_000') # prod database
-    #'sqlite_file:/afs/cern.ch/user/j/jez/public/tauID/RecoTauTag_MVAs_2014Oct22.db')
```

Comment:
did these commented out lines have to be modified?
If not, I suggest not to so that the reco signature is not needed